decompiler/VuDisasm/VuDisassembler.cpp

```diff
@@ -5,6 +5,7 @@
 #include "third-party/fmt/core.h"
 #include "common/util/print_float.h"
 #include "common/util/Assert.h"
+#include "common/log/log.h"
```

Comment:
not needed?

Reply:
(also true in a few other files)

Reply:
I wonder if we can eventually wire this up to our CI process https://github.com/include-what-you-use/include-what-you-use